pulpcore/cli/deb/publication.py

```diff
@@ -73,6 +73,7 @@ def publication(ctx: click.Context, pulp_ctx: PulpCLIContext, publication_type:
         default_plugin="deb",
         default_type="apt",
         context_table={"deb:apt": PulpSigningServiceContext},
+        href_pattern=PulpSigningServiceContext.HREF_PATTERN,
```

Comment:
Since this change is not yet released with pulp-cli, there's a question: Are we ready to bump the minimal required version here already (not until there is a release, i guess), or do we want to fudge the value here for a while with a comment to revert to this final variant once the minimal reqs allow?

Reply:
@tjmullicani Can you rebase this branch to latest main branch, and then adjust the following line:
https://github.com/pulp/pulp-cli-deb/blob/main/setup.py#L30
to use `"pulp-cli>=0.19.0"`?

@mdellweg That is what we need to do here, right? And then we can merge once `pulp-cli` 0.19.0 has been released?

Reply:
Yes, that would work.
Alternatively, something like `getattr(PulpSigningServiceContext, "HREF_PATTERN", "/signing_service/")` until we really bump that dependency. I just fear that kind of a workaround would possibly be forgotten to remove eventually.